third_party/move/move-binary-format/src/deserializer.rs

```diff
@@ -87,6 +87,12 @@ impl DeserializerConfig {
     }
 }
 
+impl Default for DeserializerConfig {
+    fn default() -> Self {
+        Self::new(VERSION_MAX, IDENTIFIER_SIZE_MAX)
```

Comment:
Actually changed to `VERSION_MAX`, as this what was used in `VMConfig::default()`... also added a comment about this. I am pretty sure this is not correct and should be default, because MAX is 7...

In any case, using `VERSION_MAX` preserved the old behaviour for now.

Reply:
@vgao1996 @zekun000 @junkil-park do we really want version max to be the default? This is what the old code was doing, so double-checking.